CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
```

Comment:
What changed?

Reply:
I don't think its super important to list that here. People can look at the PR description if they're interested